python/tvm/rpc/client.py

```diff
@@ -160,7 +160,12 @@ def load_module(self, path):
         m : Module
             The remote module containing remote function.
         """
-        return _ffi_api.LoadRemoteModule(self._sess, path)
+        module = _ffi_api.LoadRemoteModule(self._sess, path)
+        type_key = self.get_function("runtime.ModuleGetTypeKey")(module)
+        if type_key == "GraphRuntimeFactory":
+            from tvm.runtime.graph_runtime_factory import GraphRuntimeFactoryModule
+            return GraphRuntimeFactoryModule(module)
```

Comment:
I prefer `GraphRuntimeFactoryModule`, because this `GraphRuntimeFactoryModule` doesn't is a graph runtime, instead it will create different graph runtime (vm) according to different options.

Reply:
Thanks! I totally agree with you :)
It is just that in usual practice any sort of Factory the Naming follows convention of XXXFactory. But here it is little different.